Rajitha/itemPrice.js

```diff
@@ -0,0 +1,36 @@
+/**********************************************************************
+ * Create a function that takes an array of objects (groceries) which 
+ * calculates the total price and returns it as a number. A grocery 
+ * object has a product, a quantity and a price,
+ ***********************************************************************/
+const getItemPrice = arr => {
+  let iArr = arr;
+  let itemPrice = 0;
+ 
+  for (let i = 0; i < iArr.length; i++){
+    itemPrice += Number(iArr[i].price) * (iArr[i].quantity);
```

Comment:
hmm why are you using Number here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number